packages/lucide-preact/src/createPreactComponent.js

```diff
@@ -24,7 +24,7 @@ export default (iconName, iconNode) => {
         class: `lucide lucide-${toKebabCase(iconName)}`,
         ...rest,
       },
-      iconNode.map(([tag, attrs]) => h(tag, attrs)),
+      [...toChildArray(children), ...iconNode.map(([tag, attrs]) => h(tag, attrs))],
```

Comment:
Is it not better to spread the children after the iconChildren, so we can render things on top of the icon?
Like I discussed this feature in #477

Reply:
That depends. As you can see from the commits, that was my initial idea as well. MDN however recommends putting `<title>` elements (which were my primary motivation to add this feature) at the top of `<svg>` elements, as this [is required by spec in SVG 1.1](https://www.w3.org/TR/SVG11/struct.html#TitleElement), otherwise browsers will not show tooltips:

```
In conforming SVG document fragments, any ‘title’ element should be the first child element of its parent. Note that those implementations that do use ‘title’ to display a tooltip often will only do so if the ‘title’ is indeed the first child element of its parent.
```

I do believe however, this is no longer the case in the newer iterations of SVG (but haven't been able to confirm this).

Reply:
Thanks for the extensive testing! If it's this widely supported, I think the benefits of icon customization options outweigh the backward compatibility with an old SVG standard - especially taking into account it's a new feature to this package so it doesn't break existing implementations depending on it.

I don't know if you're able to do so, otherwise I'll revert the latest commit so it's in the proposed order tomorrow (GMT+1).

Reply:
@MoltenCoffee Thanks ! Tomorrow is fine, no hurry 😄

Reply:
Done!